### ISSUE TITLE
[Rust-Server] it generates broken codes in some cases around mimetypes

### ISSUE THREAD

**Issue by KeenS** [swagger-api/swagger-codegen#7126]
##### Description

when the mime type is not a single token, like `application/vnd.github.v3+json`, it produces invalid rust code

##### Swagger-codegen version

e75523a6846f0aa584ada33674744f395b781a7f 

##### Swagger declaration file content or url

https://gist.github.com/KeenS/195f694ce0e0215dc76541f799ba713d

##### Command line used for generation

```
$ java -jar wagger-codegen-cli.jar generate  -v -l rust-server -i swagger.json   -o server
```

##### Steps to reproduce

run the command above and run

##### Related issues/PRs

None

##### Suggest a fix/enhancement

use `"mime/type".parse().unwrap()`. I'll do it.

**Comment by wing328** [comment]
@KeenS thanks for reporting the issue. Let us know if you need help with the PR.

**Comment by KeenS** [comment]
It's ready but waiting for #7125 because they conflicts. Thank you.

**Comment by KeenS** [comment]
Fixed in #7212
